common/src/project.rs

```diff
@@ -115,18 +116,26 @@ impl TryFrom<Url> for Origin {
 pub struct PeerIdentity {
     pub urn: Urn,
     pub name: String,
+    pub ens: Option<String>,
```

Comment:
We should keep this as `Ens`, there may be more things we want to include in the ens payload in the future than the name.

Reply:
Okay no problem 👍